aws_lambda_builders/workflows/custom_make/workflow.py

```diff
@@ -44,10 +47,6 @@ def __init__(self, source_dir, artifacts_dir, scratch_dir, manifest_path, runtim
 
         subprocess_make = SubProcessMake(make_exe=self.binaries["make"].binary_path, osutils=self.os_utils)
 
-        # Don't build in source by default (backwards compatibility)
-        if build_in_source is None:
-            build_in_source = False
-
```

Comment:
I recently added this in a previous PR to handle build-in-source defaults for custom makefile, but it's not needed anymore with the new class constant logic (`BUILD_IN_SOURCE_BY_DEFAULT`)